native/fhir-to-bal-lib/src/main/java/org/wso2/healthcare/fhir/ballerina/packagegen/tool/modelgen/versions/r4/R4ResourceContextGenerator.java

```diff
@@ -201,6 +203,24 @@ public int compare(ElementDefinition e1, ElementDefinition e2) {
                     isSlice = true;
                 }
             }
+
+            if(elementDefinition.hasContentReference()){
```

Comment:
```suggestion
            if(elementDefinition.hasContentReference()) {
```
Add formatter for the newly added code